lib/modules/asset/AssetHistoryService.ts

```diff
@@ -29,23 +28,33 @@ export class AssetHistoryService {
 
     onAsk<AskAssetHistoryAdd<AssetHistoryEvent>>(
       "ask:device-manager:asset:history:add",
-      async ({ engineId, event, asset }) => this.add(engineId, event, asset)
+      async ({ engineId, histories }) => this.add(engineId, histories)
     );
   }
 
   async add<TAssetHistoryEvent extends AssetHistoryEvent>(
     engineId: string,
-    event: TAssetHistoryEvent,
-    asset: KDocument<AssetContent>
+    histories: AssetHistoryContent[]
   ) {
-    await this.sdk.document.create<AssetHistoryContent<TAssetHistoryEvent>>(
+    const contents: mCreateRequest<
+      AssetHistoryContent<TAssetHistoryEvent, any, any>
+    > = [];
+
+    for (const { asset, event, id, timestamp } of histories) {
+      contents.push({
+        body: {
+          asset,
+          event,
+          id,
+          timestamp,
+        },
+      });
+    }
+
+    await this.sdk.document.mCreate<AssetHistoryContent<TAssetHistoryEvent>>(
       engineId,
       InternalCollection.ASSETS_HISTORY,
-      {
-        asset: asset._source,
-        event,
-        id: asset._id,
-      }
+      contents
```

Comment:
Explicitely throw if an error occur
```suggestion
      contents,
      { strict: true }
```